packages/react-component-library/src/components/AutocompleteE/AutocompleteEOption.tsx

```diff
@@ -3,11 +3,28 @@ import React from 'react'
 import { SelectBaseOption, SelectBaseOptionProps } from '../SelectBase'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AutocompleteEOptionProps extends SelectBaseOptionProps {}
+export interface AutocompleteEOptionProps extends SelectBaseOptionProps {
+  inputValue?: string
+}
+
+function emboldenInputValue(children: string, inputValue: string) {
+  if (inputValue) {
+    const regex = new RegExp(`(${inputValue})`, 'gi')
```

Comment:
The RegExp is searching anywhere in the string while the actual search is only searching from the beginning. That leads to e.g.:

![image](https://user-images.githubusercontent.com/66470099/149375922-f80c8491-e083-49b2-a87b-46154056da16.png)

(the second 't' in thirty is bold).

Would be good to make the two consistent.

Reply:
Actually there is a bug with the actual filter because it should search anywhere to reach feature parity with the existing `Select` so it's a good time to fix that bug.